k0764427/Assignment7.py

```diff
@@ -0,0 +1,18 @@
+from sys import argv
+
+script, text, surround = argv
```

Comment:
Not that it isn't necessary, but that you are _unpacking_ it into 3 elements before verifying how many elements you have.